RecoJets/JetProducers/interface/VirtualJetProducer.h

```diff
@@ -0,0 +1,223 @@
+#ifndef newVirtualJetProducer_h
+#define newVirtualJetProducer_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h